zebra/zebra_nhg.c

```diff
@@ -450,12 +450,13 @@ static void *zebra_nhg_hash_alloc(void *arg)
 					 nhe->nhg.nexthop->vrf_id);
 		if (ifp)
 			zebra_nhg_set_if(nhe, ifp);
-		else
-			flog_err(
-				EC_ZEBRA_IF_LOOKUP_FAILED,
```

Comment:
Looks like this code may be unused after this - could it be removed too?

Reply:
Its hard to tell what code you are marking in the github UI.  If you mean `zebra_nhg_set_if()`, that's still being used.

Reply:
heh - sorry:  I mean the error code: `EC_ZEBRA_IF_LOOKUP_FAILED`

Reply:
yep yep, removed.